install/install.py

```diff
@@ -93,7 +109,19 @@ def install(self, cmd_arguments):
         cp = ConfigParser.ConfigParser({"RootDir":self.RootDir, "Pid":self.pid})
         cp.read(self.distros_cfg)
 
-        #Try get the distro automatically
+        # Check if the current distribution is compatible
+        
+
+        if not self.is_compatible():
+            Colorizer.danger("\n[!] You don't seem to have the apt package manager installed. "
+                "This means that your distribution is not fully supported (yet).")
+
+            # yes is assumed for any other input
+            response = raw_input("\nContinue installation anyway? [Y/N] : ")
+            if(response == 'N' or response == 'n'):
```

Comment:
Ditto. Also just call `response.lower() or response.upper()` to do case-insensitive comparison.